src/ParametersHandler/include/BipedalLocomotionControllers/ParametersHandler/IParametersHandler.h

```diff
@@ -27,6 +27,10 @@ template <class Derived> class IParametersHandler
 
     using unique_ptr = std::unique_ptr<IParametersHandler<Derived>>;
```

Comment:
This one should not be required anymore

Reply:
I did not remove the ``unique_ptr`` logic. You are still free to own the handler with a ``unique_ptr``. The issue was only in the way the group was returned.